pyg_lib/ops/__init__.py

```diff
@@ -227,12 +227,44 @@ def sampled_div(
     return out
 
 
+def index_sort(input: LongTensor,
+               max: Optional[int] = None) -> Tuple[LongTensor, LongTensor]:
+    r"""Sorts the elements of the :obj:`input` tensor in ascending order by
+    value. It is expected that :obj:`input` tensor is 1-dimensional and
+    contains only positive, integer values. If :obj:`max` is given, it can be
+    used by the underlying algorithm for better performance.
+
+    .. note::
+
+        This operation is optimized only for tensors associated with the CPU
+        device.
+
+    Args:
+        input (torch.LongTensor): 1-dimensional tensor with positive integer
+            values.
+        max: (int, optional): A maximum value stored inside :obj:`input`.
```

Comment:
```suggestion
        max (int, optional): A maximum value stored inside :obj:`input`.
```
Can we also rename this to `max_value` since `max` is a reserved keyword in Python?